### PR TITLE
[artifactory] Document Artifactory node affinity

### PR DESCRIPTION
**Pull request by rpf3**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
Document an undocumented configurable chart value.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by chukka** [comment]
@rpf3 Thanks for contribution. Can you please add the same change for `artifactory-ha` chart

**Comment by rpf3** [comment]
@chukka all set

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -1436,6 +1436,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.tomcat.connector.maxThreads`         | The max number of connections to Artifactory connector   | `200` |
 | `artifactory.tomcat.connector.extraConfig`         | The max queue length for incoming connections to Artifactory connector  | `'acceptCount="100"'` |
 | `artifactory.systemYaml`                      | Artifactory system configuration (`system.yaml`) as described here - https://www.jfrog.com/confluence/display/JFROG/Artifactory+System+YAML  | `see values.yaml`    |
+| `artifactory.affinity` | Artifactory node affinity | `{}` |
```
Comment (chukka) [inline stable/artifactory-ha/README.md:1439]: @rpf3 it should be `artifactory.primary. affinity` and `artifactory.node. affinity` 
please refer [primary](https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/values.yaml#L1126) and [node](https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/values.yaml#L1184)

**Comment by rpf3** [comment]
@chukka I've made the changes but the rebasing and history are a little bit of a mess now.  Should I open a new PR?

**Comment by chukka** [comment]
> @chukka I've made the changes but the rebasing and history are a little bit of a mess now. Should I open a new PR?

Looks ok , not needed

**Pull request merged**